apache-activemq-mixin/README.md

```diff
@@ -0,0 +1,103 @@
+# Apache ActiveMQ mixin
+
+The Apache ActiveMQ mixin is a set of configurable Grafana dashboards and alerts.
+
+The Apache ActiveMQ mixin contains the following dashboards:
+
+- Apache ActiveMQ cluster overview
+- Apache ActiveMQ instance overview
+- Apache ActiveMQ queue overview
+- Apache ActiveMQ topic overview
+- Apache ActiveMQ logs overview
+
+and the following alerts:
+
+- ApacheActiveMQHighTopicMemoryUsage
+- ApacheActiveMQHighQueueMemoryUsage
+- ApacheActiveMQHighStoreMemoryUsage
+- ApacheActiveMQHighTemporaryMemoryUsage
+
+## Apache ActiveMQ cluster overview
+
+The Apache ActiveMQ cluster overview provides cluster level statistics showing cluster count, broker count, producer counts, consumer counts, overall memory level usage, and a general look into enqueue/dequeue counts.
+![Screenshot of the Apache ActiveMQ cluster overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/aerospike/screenshots/apache_activemq_cluster_overview.png)
+
+## Apache ActiveMQ instance overview
+
+The Apache ActiveMQ instance overview provides instance level statistics showing memory levels, producer counts, consumer counts, queue sizes, enqueue/dequeue counts, average enqueue times, expired message counts, jvm garabage collection info, and alerts.
+![First screenshot of the Apache ActiveMQ instance overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/aerospike/screenshots/apache_activemq_instance_overview_1.png)
+![Second screenshot of the Apache ActiveMQ instance overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/aerospike/screenshots/apache_activemq_instance_overview_2.png)
+
+## Apache ActiveMQ queue overview
+
+The Apache ActiveMQ queue overview provides queue level statistics showing number of queues, message size, producer counts, consumer counts, enqueue/dequeue rates, average enqueue times, expired message rates, and average size of messages.
```

Comment:
```suggestion
The Apache ActiveMQ queue overview provides queue level statistics showing number of queues, message size, producer counts, consumer counts, enqueue/dequeue rates, average enqueue times, expired message rates, and average message size.
```
this applies for the other descriptions too, but super nit unless there's a difference between what you have and this suggestion... your call ultimately

Reply:
I prefer the first.